src/StarRatings/StarRating.razor

```diff
@@ -0,0 +1,50 @@
+﻿<div class="@CssClass" style="@($"--star-rating: {Value};")">
```

Comment:
Might not work on non US systems.
For example with a french system, a rating of 2.7 will make the style attribut to be "--star-rating: 2,7" and calculation of --percent will not work.
```suggestion
﻿<div class="@CssClass" style="@($"--star-rating: {Value.ToString(System.Globalization.CultureInfo.InvariantCulture)};")">
```
Using ToString with CultureInvariant will make it work for non US systems